packages/babel-plugin-proposal-explicit-resource-management/src/index.ts

```diff
@@ -160,6 +160,58 @@ export default declare(api => {
         }
       }
     },
+    SwitchStatement(path, state) {
+      if (!process.env.BABEL_8_BREAKING && !state.availableHelper("usingCtx")) {
+        return;
+      }
```

Comment:
Maybe we add a warning here?

Reply:
If would prefer to check if there is a `using` statement in the switch, and if there is throw an error saying "we cannot transpile this, update `@babel/core`".